modules/apps/portal-remote/portal-remote-jaxrs-whiteboard/src/main/java/com/liferay/portal/remote/jaxrs/whiteboard/lifecycle/JAXRSLifecycleUtil.java

```diff
@@ -10,44 +10,28 @@
 
 import org.osgi.framework.BundleContext;
 import org.osgi.framework.ServiceRegistration;
-import org.osgi.service.component.annotations.Activate;
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Deactivate;
 
 /**
  * @author Stian Sigvartsen
  */
-@Component(service = JAXRSLifecycle.class)
-public class JAXRSLifecycle {
+public class JAXRSLifecycleUtil {
 
-	public void ensureReady() {
+	public static void ensureReady(BundleContext bundleContext) {
 		if (_jaxrsReady) {
```

Comment:
Hey @RenanUmV, I think, we do not need this flag to control the initialization. The DCLSingleton mechanism cover this already. Please check that and resend. Thx